tests/SpecTests/crud/updateWithPipelines.json

```diff
@@ -238,6 +238,171 @@
           ]
         }
       }
+    },
+    {
+      "description": "UpdateOne in bulk write using pipelines",
```

Comment:
Do you suppose it's worth having a single test in BulkWriteFunctionalTest.php for pipelines? It can execute a single `BulkWrite` that contains both an `updateOne` and `updateMany` each using a pipeline and preface itself with a requirement for server 4.2+.

Reply:
Good idea. I've added a functional test for this as well.